reachy_tictactoe/tictactoe_playground.py

```diff
@@ -354,11 +361,14 @@ def is_final(self, board):
 
     def has_human_played(self, current_board, last_board):
         cube = piece2id['cube']
+        
+        delta = current_board - last_board 
 
-        return (
-            np.any(current_board != last_board) or
-            np.sum(current_board == cube) > np.sum(last_board == cube)
-        )
+        return np.sum(delta != 0) == 1 and np.sum(delta) == 2 
```

Comment:
Maybe doing it like this will prevent the cheating detection if the human plays multiple cubes.